src/components/Card.js

```diff
@@ -39,9 +51,21 @@ export default class Card {
     const cardImage = this._cardElement.querySelector(".card__image");
     const cardCap = this.name;
     const cardImg = this.link;
+    const cardID = this._id;
     cardName.textContent = cardCap;
     cardImage.src = cardImg;
     cardImage.alt = cardCap;
+    this._cardElement.id = cardID;
+    this._cardElement.isLiked = this.isLiked;
+    if (this._cardElement.isLiked) {
+      this._cardElement
+        .querySelector(".card__button-like")
```

Comment:
**Could be improved**

All elements found **every time** you call a method should be defined as **class fields** in the constructor, `_setEventListeners` (or in `getView`) only **1 time** in order not to waste resources on searching them again and again.

`Class fields` (the so-called **class variables** with `this`) are accessible inside any class method. You shouldn’t pass **class fields** into the call of any method or search them again because you can use class fields inside all methods.
There can be `this._inputs`, `this._submitButton`, `this._cardImage`, `this._form`, `this._likeButton`, and so on.

```js
  getView() {
    this._element = this._getTemplate();
    this._likeButton = this._element.querySelector(".card__like-button");
    this._cardImage = this._element.querySelector(".card__image");
    this._trashButton = this._element.querySelector(".card__trash-button");

```